src/coq_rules.ml

```diff
@@ -111,6 +111,16 @@ let create_ccoq sctx ~dir =
   ; coqpp  = rr "coqpp"
   }
 
+(* get_libraries from Coq's ML dependencies *)
+let libs_of_coq_deps ~loc ~scope libs =
+  let lib_db = Scope.libs scope in
+  match Lib.DB.find_many ~loc lib_db
+          (List.concat_map ~f:Dune_file.Lib_dep.to_lib_names libs) with
+  | Ok libs ->
+    libs
+  | Error exn ->
+    raise exn
```

Comment:
You can simply use `Result.ok_exn`.

In fact, something like this would look clean I think:

```ocaml
List.concat_map ~f:Dune_file.Lib_dep.to_lib_names libs
|> Lib.DB.find_many ~loc lib_db
|> Result.ok_exn
```

Reply:
Indeed, thanks @rgrinberg , PR updated.